test/only.js

```diff
@@ -0,0 +1,13 @@
+QUnit.module( "QUnit.only" );
+
+QUnit.test( "implicitly skipped test", function( assert ) {
+	assert.ok( false, "test should be skipped" );
+});
+
+QUnit.only( "only run this test", function( assert ) {
+	assert.ok( true, "only this test should run" );
+});
+
+QUnit.test( "another implicitly skipped test", function( assert ) {
+	assert.ok( false, "test should be skipped" );
+});
```

Comment:
The consistency is a very reasonable argument. That might be simple if we register the first `only` assignment and check it on every only call. If another `only` is called, it will be registered as a regular test.